homeassistant/components/webostv/__init__.py

```diff
@@ -166,6 +166,14 @@ async def async_migrate_task(
             else:
                 break
 
+        ent_reg = entity_registry.async_get(hass)
+        if not ent_reg.async_get_entity_id(Platform.MEDIA_PLAYER, DOMAIN, key):
+            _LOGGER.debug(
+                "Not updating webOSTV Smart TV entity %s unique_id, entity unavailable",
+                entity_id,
+            )
+            return
+
         uuid = client.hello_info["deviceUUID"]
         ent_reg.async_update_entity(entity_id, new_unique_id=uuid)
```

Comment:
Maybe we need to get the entity_id again before updating the entry? The entity entry may still be there but the user can have changed the entity_id after we started the migration task.